src/exits.h

```diff
@@ -0,0 +1,117 @@
+#pragma once
+
+#include <cstdlib>
+
+#include <string>
+#include <unordered_map>
+#include <utility>
+
+#include <osmium/builder/osm_object_builder.hpp>
+#include <osmium/handler.hpp>
+#include <osmium/osm/types.hpp>
+
+#include "util.h"
+
+// Collects node ids and ref=* tags on highway=motorway_junction.
+// http://wiki.openstreetmap.org/wiki/Tag:highway%3Dmotorway_junction
+// http://taginfo.openstreetmap.org/keys/exit_to
+// http://wiki.openstreetmap.org/wiki/Proposed_features/junction_details
+struct ExitsRewriter : osmium::handler::Handler {
+  static const constexpr auto kRefsOnJunctionInPlanet = 91000u;
```

Comment:
Is hardcoding numbers of entries in the planet going to be a problem at some point, or is it only for the reservation below?